src/languages/verilog.js

```diff
@@ -76,30 +499,30 @@ export default function(hljs) {
       'svh'
     ],
     case_insensitive: false,
-    keywords: SV_KEYWORDS,
+    keywords: KEYWORDS,
     contains: [
       hljs.C_BLOCK_COMMENT_MODE,
       hljs.C_LINE_COMMENT_MODE,
       hljs.QUOTE_STRING_MODE,
       {
-        className: 'number',
+        scope: 'number',
         contains: [ hljs.BACKSLASH_ESCAPE ],
         variants: [
           {
-            begin: '\\b((\\d+\'(b|h|o|d|B|H|O|D))[0-9xzXZa-fA-F_]+)'
+            begin: /\b((\d+'(b|h|o|d|B|H|O|D))[0-9xzXZa-fA-F_]+)/
```

Comment:
```suggestion
            begin: /\b((\d+'([bhodBHOD]))[0-9xzXZa-fA-F_]+)/
```

Since we're touching this regex; might as well simplify it a tad